### PR TITLE
Implementing the All Your Base exercise

### PR DESCRIPTION
**Pull request by camilopayan**
Calling dibs on this one!

### CODE REVIEW

**Comment by camilopayan** [comment]
Ready for review!

File: exercises/all-your-base/all-your-base_test.php [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+ */
+class AllYourBaseTest extends PHPUnit\Framework\TestCase
+{
+    public function testSingleBitOneToDecimal()
+    {
+        $this->assertEquals(rebase(2, [1], 10), [1]);
+    }
+
+    /**
+     * @markTestSkipped
```
Comment (petemcfarlane) [inline exercises/all-your-base/all-your-base_test.php]: The tests look fine, I'm wondering if we should change these from annotations to be `$this->markTestSkipped()` for consistency with the other exercises - what are your thoughts @camilopayan ?

File: exercises/all-your-base/all-your-base.php [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?php
+
+function rebase(int $fromBase, array $digits, int $toBase)
```
Comment (petemcfarlane) [inline exercises/all-your-base/all-your-base.php]: It's not normally necessary to include the answer file with the exercise. The only exception to that is `bob`, which is the first exercise to get people start. So this file can be removed please 👍

File: exercises/all-your-base/all-your-base_test.php (hunk 1 above)
Comment (camilopayan) [reply exercises/all-your-base/all-your-base_test.php]: I've always used the annotation, but changing it shouldn't be a big deal.

**Pull request merged**